tools/screenshot_check/screenshot_check.py

```diff
@@ -0,0 +1,27 @@
+import os, sys
+from PIL import Image
```

Comment:
Maybe add an error in this script for when it runs without Pillow installed. A future user might skip the readme.